backend/src/main/java/ai/verta/modeldb/project/ProjectDAORdbImpl.java

```diff
@@ -919,18 +919,21 @@ public ProjectPaginationDTO findProjects(
       projectRoot.alias("pr");
       List<Predicate> finalPredicatesList = new ArrayList<>();
 
-      Set<String> accessibleProjectIds = new HashSet<>();
+      Set<String> accessibleResourceIdsWithCollaborator =
```

Comment:
I'm assuming owner counts as collaborator, right?

Reply:
I do not understand your question.  `accessibleResourceIdsWithCollaborator ` is a list of  the accessible resources ids with collaborators.

Reply:
If I own a project, will the project be included in the list?

Reply:
Yes. it is added. that list has my own project ids as well as those ids where i am a collaborator

Reply:
The name is super confusing to me. Because `accessibleResourceIdsWithCollaborator` reads as "accessible ids of resources with collaborators", which I don't think it what it's doing from the description (does it have to have a collaborator?)